### ISSUE TITLE
drag and drop between trees broken

### ISSUE THREAD

**Issue by tbnorth** [leo-editor/leo-editor#135]
Load Leo with two outlines, then detach one (assuming tabbed interface).

Expect dragging a node from one tree to the other will move it, maybe copy if Alt held.  Doesn't work, although nodes in receiving tree can be reordered sometimes.  git bisect, below, identifies offending commit. (Log info. just shows "today's" Leo, but this has been broken a long time.)
```
Leo 5.0-final, build 20150127110559, Tue, Jan 27, 2015 11:05:59 AM
Git repo info: branch = master, commit = e4b9b05e377f
Python 2.7.6, PyQt version 4.8.6
linux2

4f848037424544e127166dbb0375f90c6826aa21 is the first bad commit
commit 4f848037424544e127166dbb0375f90c6826aa21
Author: Edward K. Ream <edreamleo@gmail.com>
Date:   Fri Sep 13 16:44:33 2013 -0500

    Removed several .dragging ivars, so that multiple inter-file drags can happen.
```

**Comment by edreamleo** [comment]
Thanks for this sleuthing.  I'll fix it asap.

 I was just about to reopen a bug (iirc) about paste-retaining-clones being broken. Interestingly, I was able to drag and drop an outline when paste-retaining-clones failed to retain clones.

**Comment by edreamleo** [comment]
Fixed at rev f5eccd3.
